src/mixins/maskable.js

```diff
@@ -60,8 +60,10 @@ export default {
       let position = 0
       let selection = this.selection
 
-      for (const char of oldText.substr(0, selection)) {
-        isMaskDelimiter(char) || position++
+      if (oldText) {
+        for (const char of oldText.substr(0, selection)) {
+          isMaskDelimiter(char) || position++
+        }
```

Comment:
`oldText` will always be a string, this is redundant

Reply:
Well, that's because I  expect `null` to be a valid value, and that `this.lazyValue` has to be able to accept null.

Reply:
`oldText` is directly from the html element, which is *always* a string, even if you set it to null first.

Reply:
This is still redundant

Reply:
Indeed it is. `input.value` never becomes null. Even if it's explicitly set to null, it turns to an empty string. At least in the latest Firefox. I'm changing it right away.